tests/e2e/workloads/test_data_consistency.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf8 -*-
+
+import logging
+import os
+import time
+
+import pytest
+import yaml
+
+from ocs_ci.framework.pytest_customization import marks
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import exceptions
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.fio_artefacts import get_pvc_dict
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile, link_spec_volume
+from ocs_ci.utility.utils import run_cmd
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@marks.bugzilla("1989301")
+@pytest.mark.polarion_id("OCS-2735")
+def test_log_reader_writer_parallel(project, tmp_path):
+    """
+    Write and read logfile stored on cephfs volume, from all worker nodes of a
+    cluster via k8s Deployment, while fetching content of the stored data via
+    oc rsync to check the data locally.
+
+    Reproduces BZ 1989301. Test failure means new blocker high priority bug.
+    """
+    pvc_dict = get_pvc_dict()
+    # we need to mount the volume on every worker node, so RWX/cephfs
+    pvc_dict["metadata"]["name"] = "logwriter-cephfs-many"
+    pvc_dict["spec"]["accessModes"] = [constants.ACCESS_MODE_RWX]
+    pvc_dict["spec"]["storageClassName"] = constants.CEPHFILESYSTEM_SC
+    # there is no need for lot of storage capacity for this test
+    pvc_dict["spec"]["resources"]["requests"]["storage"] = "1Gi"
+
+    # get deployment dict for the reproducer logwriter workload
```

Comment:
I suggest to turn some comments from this file into debug/info logs. The code is nicely commented but it would be good to have some information also in test run logs and utilize it during test run analysis.

Reply:
I sprinkled few info lines (but only when "world" changing action is going to happen in a next step).